README.md

```diff
@@ -11,7 +11,7 @@ Some requirements in this alpha stage are that your Nextcloud:
 
 ![](img/screenshot.png)
 
-- **🙋 Find your friends:** No matter if they use Nextcloud, [🐘 Mastodon](https://joinmastodon.org), [🇫 Friendica](https://friendi.ca), and soon [✱ Diaspora](https://joindiaspora.com), [👹 MediaGoblin](https://www.mediagoblin.org) and more – you can follow them!
+- **🙋 Find your friends:** No matter if they use [☁ Nextcloud Social](https://github.com/nextcloud/social#nextcloud-social), [🐘 Mastodon](https://joinmastodon.org), [🇫 Friendica](https://friendi.ca), [📺 PeerTube](https://joinpeertube.org), [🅿️ Pleroma](https://pleroma.social) or [📷 PixelFed](https://pixelfed.org) – you can follow them!
```

Comment:
@daita @jancborchardt Did one of you already run a test with peertube/pleroma/pixelfed?

Reply:
tested right now with peertube, it seems that peertube does not really communicate with others ActivityPub from the fediverse

Reply:
So then:

> No matter if they use Nextcloud Social, Mastodon, Friendica – and hopefully soon PeerTube, Pleroma or Pixelfed – you can follow them!

?

Reply:
Or if it’s nearly done with a new pull request anyway, let’s just merge it?

Reply:
You should remove them and we'll keep the list up-to-date after each release